### DISCUSSION TITLE
Is this project alive?

### DISCUSSION THREAD

**Discussion by LuizDMM** [MongoEngine/django-mongoengine#175] [General]
Hey guys, how are you doing today?

I'm working with a start-up that uses Django and MongoDB, and we're currently using "Djongo" as Object-Document-Mapper, but that project seems to be using the contributions only to their paid version. The pip version hasn't received upgrades for the last couple years, besides having a lot of pushes in main.

That said, I'm looking for a solution to migrate our ODM in the next couple of months. One solution would be to clone their repo and create a new pip package, but I never did that. That said, I have some experience with Django and I can try to help a lot in my free time because it's a core future for us. But before committing to making PRs, I have to ask, is this project alive? If so, how can I help?

**Comment by ngocngoan** [comment]
It seems to be stopped due to the reason at https://github.com/MongoEngine/django-mongoengine/commit/cc7d0399516b42c69618959732f5c0b0804cc254. I am also finding a replacement solution for my library (https://github.com/ngocngoan/djangorestframework-simplejwt-mongoengine)

**Comment by last-partizan** [comment]
If you're already using Djongo and it works fine for you, i suggest continue using it.

You can install git version with pip easily. Or maybe switch to their paid version, if they're taking money, i suggest they're have time to support it properly.

This project does not receive any active updates, it's in mainternance mode and there will be no new features. But, if you're willing to fix some bugs - i will review PR's and merge them.

**Reply by LuizDMM** [reply]
Hey man, thanks for your answer. I'm working with python for 3 years now, and I didn't know it was possible to install git versions, I'll try that, thank you.

The problem is, the current version has several bugs that require a lot of work around, so it's no really working fine. And it doesn't seem to be receiving a lot of support, because there are old PRs fixing some bugs that were never reviewed.

We'd happily pay for a commercial license, but the question is that their paid version doesn't support keeping our code on our on servers, and all our code is in a kubernetes cluster. Also, there are some complaints in their [group](https://groups.google.com/g/djongo/c/k1vTy3XA9Gk) of users that can't even register.

I'll take a look at your code, and I'll really try to help as I can.

**Reply by last-partizan** [reply]
Maybe raw mongoengine can work for you.

This project is mostly wrappers and monkey-patches to get things working with django forms and admin, but really what you need most is just model definitions and ways to query them, and `mongoengine` already gives you all this.

**Reply by LuizDMM** [reply]
Thank you! I'll take a look at it as well. It maybe would take some work to integrate, but it may be worth.

**Comment by last-partizan** [comment]
@ngocngoan for your case, i can suggest using raw mongoengine? Do you think you can use it?

**Comment by ngocngoan** [comment]
@last-partizan Yes, I can use it. But, my library is using the MongoUser document as the reference field. So, I need to replace it before removing this dependency
